.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/lts
+
+    working_directory: ~/repo
```

Comment:
what is repo? @piyushagwl007

Reply:
Some default folder in which we need to run the commands